### ISSUE TITLE
`undefined` in `plugins` array causes Hyper to refuse to open windows, with no visible errors whatsoever

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [zeit/hyper#1862]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS Sierra **10.12.5 Beta** (16F60a)
- **Hyper.app version**: Hyper ** 1.3.3.1754**
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/ELLIOTTCABLE/678ad79704a104c8f29efcfaddd09e07
- **The issue is reproducible in vanilla Hyper.app**: **Yes**

## Issue
Having accidentally commented out the first element of a comma-prefixed `plugins` array in `.hyper.js`,

```js
   plugins: [
  //  'hypercwd'
    , 'hyperterm-alternatescroll'
    , 'hyperterm-focus-reporting'
      // ...
   ],
```

… I discovered that Hyper shows no errors, but refuses to open any windows.

Probably not a common mistake, or a huge problem, but I thought it'd be worth reporting. 😜

**Comment by ppot** [comment]
#1509

**Comment by nitsakh** [comment]
Hello, I would like to take up this issue. I haven't contributed to Hyper before. Would the #hyper channel on slack be a good place to get help?

**Comment by chabou** [comment]
Do not hesitate to ask on #hyper channel.
Yep, we must ignore `undefined` plugins. Maybe remove `undefined`entries in plugin array after config parsing.

**Comment by albinekb** [comment]
😂 comma prefixed, nice 👌 

Yeah let's just do `.filter(Boolean)`

**Comment by albinekb** [comment]
Fixed in #1927

**Comment by ELLIOTTCABLE** [comment]
<'3

Thanks, @nitsakh! And welcome to open-source!
